static/docs/commands-reference/status.md

```diff
@@ -50,21 +50,45 @@ detected, `dvc status` prints this message:
 ```
 
 This says that no differences were detected, and therefore that no stages would
-be run again if `dvc repro` were executed.
+be run again by `dvc repro`.
```

Comment:
- [ ] `This says` -> `This indicates`.
- [ ] `therefore that no` -> `therefore no`

Reply:
amended, kindly review these changes.